DNN Platform/Library/Entities/Tabs/TabVersions/TabVersionController.cs

```diff
@@ -42,17 +43,21 @@ public TabVersion GetTabVersion(int tabVersionId, int tabId, bool ignoreCache =
         
         public IEnumerable<TabVersion> GetTabVersions(int tabId, bool ignoreCache = false)
         {
-            //if we are not using the cache, then remove from cacehh and re-add loaded items when eeded later
+            var user = UserController.Instance.GetCurrentUserInfo();
+            var userTimeZone = user.Profile.PreferredTimeZone;
+
+            //if we are not using the cache, then remove from cache and re-add loaded items when eeded later
             var tabCacheKey = GetTabVersionsCacheKey(tabId);
             if (ignoreCache || Host.Host.PerformanceSetting == Globals.PerformanceSettings.NoCaching)
             {
                 DataCache.RemoveCache(tabCacheKey);
             }
-            
-            return CBO.GetCachedObject<List<TabVersion>>(new CacheItemArgs(tabCacheKey,
+            var tabVersions = CBO.Instance.GetCachedObject<List<TabVersion>>(new CacheItemArgs(tabCacheKey,
                                                                     DataCache.TabVersionsCacheTimeOut,
                                                                     DataCache.TabVersionsCachePriority),
-                                                            c => CBO.FillCollection<TabVersion>(Provider.GetTabVersions(tabId)));            
+                                                            c => CBO.FillCollection<TabVersion>(Provider.GetTabVersions(tabId)), false);
```

Comment:
What's the purpose of this `false` argument @mean2me ?

Reply:
Previously no value was passed, which is equivalent to `true`. Is this change intentional?